gulp-tasks/gulp-css.js

```diff
@@ -20,6 +20,13 @@
   var del = require('del');
 
   module.exports = function (gulp, config, tasks, browserSync) {
+    
```

Comment:
You OK if we remove these dest_vue references or were those purposeful with this PR?